site/specs-temp/numbers.json

```diff
@@ -17816,7 +17819,260 @@
         "required": [
           "eventType"
         ]
+      },
+      "AccountResponse": {
+        "type": "object",
+        "properties": {
+          "Account": {
+            "$ref": "#/components/schemas/Account"
+          },
+          "AddressErrorDescription": {
+            "$ref": "#/components/schemas/AddressErrorDescription"
+          }
+        }
+      },
+      "Address" : {
+        "type" : "object",
+        "properties" : {
+          "HousePrefix" : {
+            "type" : "string"
+          },
+          "HouseNumber" : {
+            "type" : "string"
+          },
+          "HouseSuffix" : {
+            "type" : "string"
+          },
+          "PreDirectional" : {
+            "type" : "string"
+          },
+          "StreetName" : {
+            "type" : "string"
+          },
+          "StreetSuffix" : {
+            "type" : "string"
+          },
+          "PostDirectional" : {
+            "type" : "string"
+          },
+          "AddressLine2" : {
+            "type" : "string"
+          },
+          "City" : {
+            "type" : "string"
+          },
+          "RegionId" : {
+            "type" : "integer",
+            "format" : "int32",
+            "deprecated": true
+          },
+          "StateCode" : {
+            "type" : "string"
+          },
+          "Zip" : {
+            "type" : "string"
+          },
+          "PlusFour" : {
+            "type" : "string"
+          },
+          "County" : {
+            "type" : "string"
+          },
+          "Country" : {
+            "type" : "string"
+          },
+          "LegacyAddressId" : {
+            "type" : "string",
+            "deprecated": true
+          },
+          "LegacyEntityId" : {
+            "type" : "string",
+            "deprecated": true
+          },
+          "ResetAddressFields" : {
+            "type" : "boolean"
+          },
+          "Latitude" : {
+            "type" : "string"
+          },
+          "Longitude" : {
+            "type" : "string"
+          },
+          "AddressType" : {
```

Comment:
Can we declare this enum as its own schema?

Reply:
See the `Reusable Enums` section [here](https://swagger.io/docs/specification/data-models/enums/) 

reason being is that the client generators function a little more nicely with the resuable enums

Reply:
Good point!